src/renderer/pages/settings-page.component.tsx

```diff
@@ -137,6 +144,25 @@ export function SettingsPage() {
         i18nService.setLanguage(item.value);
     };
 
+    const handleVersionClick = async () => {
+        let isChangelogResolved = false;
+
+        await autoUpdater.showChangelog(await lastValueFrom(autoUpdater.getAppVersion()))
```

Comment:
I think you should move this just after the setTimeout 'cause once we are at the setTimeout, the promise is already finished (Beacause it was awaited)